### ISSUE TITLE
Phase and power of reflected signal

### ISSUE THREAD

**Issue by dewi-ny-je** [jdesbonnet/RCWL-0516#13]
Is it possible to obtain phase and power of the reflected signal from some of the pins, with additional circuitry?
I'd like to use both for additional experiments, but it's not clear to me which signals I should use.

**Comment by Joerg-rw** [comment]
no, the reflected signal gets mixed down in the single-transistor-transmittter/receiver and the resulting demodulated signal doesn't yield info on those parameters

**Comment by PLaToniC48** [comment]
Can we get the signal at ant terminal befored mixer?
